force-app/main/default/lwc/settingsSaveCanvas/settingsSaveCanvas.html

```diff
@@ -1,37 +1,33 @@
 <template>
-    <div class="slds-grid slds-wrap">
-        <div class="slds-col slds-text-heading_small">
-            <span>{componentTitle}</span>
-        </div>
+    <div class="slds-grid slds-grid_vertical-align-start">
```

Comment:
It's good to feel comfortable updating this component now that we're hitting real styling. We should use its slot!